services/management/ia/rules_service.go

```diff
@@ -321,8 +321,12 @@ func (s *RulesService) CreateAlertRule(ctx context.Context, req *iav1beta1.Creat
 		For:          req.For.AsDuration(),
 		Severity:     models.Severity(req.Severity),
 		CustomLabels: req.CustomLabels,
-		ChannelIDs:   req.ChannelIds,
 	}
+
+	if len(req.ChannelIds) > 0 {
+		params.ChannelIDs = req.ChannelIds
+	}
```

Comment:
Why we need this condition? We have the same check in `models.CreateRule`

Reply:
removed redundant condition